setup.py

```diff
@@ -5,16 +5,17 @@
 
 setup(
     name='keras-trainer',
-    version='0.0.22',
+    version='0.0.23',
     description='A training abstraction for Keras models.',
     author='Triage Technologies Inc.',
     author_email='ai@triage.com',
     url='https://www.triage.com/',
     packages=find_packages(exclude=['tests', '.cache', '.venv', '.git', 'dist']),
     install_requires=[
-        'Keras==2.1.4',
+        'Keras>=2.1.6',
         'h5py',
         'Pillow',
-        'keras-model-specs>=0.0.16',
+        'tensorflow>=1.6.0',
```

Comment:
Don't add `tensorflow` as a dependency here, because then it makes it impossible to use `tensorflow-gpu` with this package